offroad/android/app/build.gradle

```diff
@@ -170,6 +170,7 @@ repositories {
 }
 
 dependencies {
+    implementation project(':@sentry_react-native')
```

Comment:
can you remove the changes with these new sentry references? I bet these were automatically added by `react-native link` in build.sh. This doesn't happen for me on macOS so I'll see if I can reproduce and fix in an ubuntu environment.

Reply:
kk fixed